src/AspNet.Security.OAuth.VkId/VkIdAuthenticationDefaults.cs

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+namespace AspNet.Security.OAuth.VkId;
+
+/// <summary>
+/// Default values used by the VK ID authentication middleware.
+/// </summary>
+public static class VkIdAuthenticationDefaults
+{
+    /// <summary>
+    /// Default value for <see cref="AuthenticationScheme.Name"/>.
+    /// </summary>
+    public const string AuthenticationScheme = "VK ID";
+
+    /// <summary>
+    /// Default value for <see cref="AuthenticationScheme.DisplayName"/>.
+    /// </summary>
+    public const string DisplayName = "VK ID";
+
+    /// <summary>
+    /// Default value for <see cref="AuthenticationSchemeOptions.ClaimsIssuer"/>.
+    /// </summary>
+    public const string ClaimsIssuer = "VK ID";
+
+    /// <summary>
+    /// Default value for <see cref="RemoteAuthenticationOptions.CallbackPath"/>.
+    /// </summary>
+    public const string CallbackPath = "/signin-vkid";
+
+    /// <summary>
+    /// Default value for <see cref="OAuthOptions.AuthorizationEndpoint"/>.
+    /// </summary>
+    public const string AuthorizationEndpoint = "https://id.vk.com/authorize";
+
+    /// <summary>
+    /// Default value for <see cref="OAuthOptions.TokenEndpoint"/>.
+    /// </summary>
+    public const string TokenEndpoint = "https://id.vk.com/oauth2/auth";
+
+    /// <summary>
+    /// Default value for <see cref="OAuthOptions.UserInformationEndpoint"/>.
+    /// </summary>
+    public const string UserInformationEndpoint = "https://id.vk.com/oauth2/user_info";
```

Comment:
These should be `static readonly` like in our other providers so that if they ever need to change (there is past precedent for other providers), the values aren't "burned in" to any call sites:

https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers/blob/82d20e3a415be2203257ecc1222edde5766961bb/src/AspNet.Security.OAuth.Amazon/AmazonAuthenticationDefaults.cs#L34-L47